### ISSUE TITLE
Code is not reloaded (?) in 0.8.0

### ISSUE THREAD

**Issue by sirmike** [hanami/hanami#634]
I'm little confused regarding code reloading. When I run a server by `bundle exec hanami server`, assets are reloaded but my code does not. When I run `bundle exec shotgun`, code seems to be reloaded but assets are stale.

Shotgun is present in Gemfile and it seems to be used by [server.rb](https://github.com/hanami/hanami/blob/v0.8.0/lib/hanami/commands/server.rb#L31) I use Hanami `v0.8.0` Is there a known bug or I am missing something important?

**Comment by jodosha** [comment]
@sirmike Hey, I'm sorry about this problem, but `v0.8.0` introduced a regression that we already fixed https://github.com/hanami/hanami/pull/628.

In a few days we'll release `v0.8.1` which will ship with the fix. Sorry for the problem.

**Comment by sirmike** [comment]
Great, I will use a patch till then. Thank you for the info. 👍

**Comment by sirmike** [comment]
I am not sure if it's related. Checked out on master and code is indeed reloaded, but assets are not refreshed on change. Will try to investigate.

**Comment by jodosha** [comment]
@sirmike Please let me know as soon you find something relevant. Thanks!

**Comment by sirmike** [comment]
It's been quite some time since I encountered this but wanted to let you know what happened.

I was fiddling with my project and it seems that it must have been something related to migration between versions. After having some problems to successfully migrate to 0.9.x I finally generated a new project, then moved all of my views / controllers there. I don't observe assets & code reloading problems anymore.
